images/kube-fluentd-operator/tests/smoke-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail -x
+
+sleep 5
+
+# There are docs to test a deployment of the operator, but this is not
+# working, same result with the upstream image. # https://github.com/vmware/kube-fluentd-operator#try-it-out
+
+# Instead, we will test the operator by deploying the log-router helm chart
+# and check the deamonset pod is healthy + the logs contain the expected text
```

Comment:
The helm_release will wait for its resources to become healthy already, FWIW. I think it's still useful to check the logs though.